Tests/SentryTests/Integrations/SentryCrash/SentryCrashIntegrationTests.swift

```diff
@@ -38,7 +38,11 @@ class SentryCrashIntegrationTests: XCTestCase {
         }
         
         func getSut() -> SentryCrashIntegration {
-            return SentryCrashIntegration(crashAdapter: sentryCrash, andDispatchQueueWrapper: dispatchQueueWrapper)
+            return getSut(crashWrapper: sentryCrash)
```

Comment:
Is "Sut" a word, or an acronym for e.g. "state under test"?

Reply:
System under test or subject under test. Basically the class you are testing.